dev/Makefile

```
@@ -127,3 +127,15 @@ review:
 delete:
 	kubectl delete namespace $(ns) --force
 
+helm-dep-updates:
+	cd ../charts && \
+	cd bria && helm dependency update && cd .. && \
+	cd dealer && helm dependency update && cd .. && \
+	cd galoy && helm dependency update && cd .. && \
+	cd galoy-deps && helm dependency update && cd .. && \
+	cd galoy-pay && helm dependency update && cd .. && \
+	cd lnd && helm dependency update && cd .. && \
+	cd loop && helm dependency update && cd .. && \
+	cd monitoring && helm dependency update && cd .. && \
+	cd rtl && helm dependency update && cd .. && \
+	cd stablesats && helm dependency update
```

```suggestion
    for dir in $$(ls ../charts); do \
		cd ../charts/$$dir && helm dependency update && cd -; \
	done
```

Couldn't we just iterate over all dirs in `/charts`?

---

much better and more future proof! (pushing the change because the Makefile needs tabs)